terraform/aws/modules/rds-aurora/main.tf

```diff
@@ -0,0 +1,101 @@
+resource "aws_rds_cluster" "this" {
+  cluster_identifier              = "${local.stack}-${var.rds_suffix}"
+  engine                          = var.db_engine_type
+  engine_version                  = var.db_engine_version
+  engine_mode                     = var.db_engine_mode
+  database_name                   = var.database_name
+  master_username                 = var.master_username
+  master_password                 = random_password.master_password.result
+  final_snapshot_identifier       = local.snapshot_name
+  skip_final_snapshot             = false
+  backup_retention_period         = var.backup_retention_period
+  preferred_backup_window         = var.backup_window
+  preferred_maintenance_window    = var.maintenance_window
+  port                            = 3306
```

Comment:
can we create this as a variable with a default? otherwise, let's hard-code the engine argument as well (instead of var.db_engine_type) and rename the module to rds-aurora-mysql.

Reply:
If you feel it's best to parameterize the port, make sure that you include that same variable in the security group!

Reply:
yes, I have created variables with default values.